spark/tests/test_spark.py

```diff
@@ -563,6 +563,7 @@ def raise_for_status(self):
 SPARK_METRIC_TAGS = ['cluster_name:' + CLUSTER_NAME, 'app_name:' + APP_NAME]
 
 
+@pytest.mark.unit
```

Comment:
Do we need to mark all tests now?

Reply:
This is because of this change https://github.com/DataDog/integrations-core/pull/4189/files#diff-250837524db7583908ad43bcf046a178R20
And `not integration` -> `unit` is needed for e2e not to be started